src/components/NavBar.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+
+function NavBar({navLinks, style}) {
+
+    const linksToDisplay = Object.entries(navLinks).map(([href, label]) => {
+        return (
+            <li key={href}>
+                <a href={href}>{label}</a>
+            </li>
+        )
+    })
+
+  return (
+    <nav>
+        <ul className={style.nav}>
+            {linksToDisplay}
+        </ul>
+    </nav>
+  )
+}
+
+  NavBar.propTypes = {
+    navLinks : PropTypes.object.isRequired,
+    style: PropTypes.object
+  };
+
+
+
```

Comment:
nit: I'm surprised the linter doesn't pick this extra spacing here. If it's not a rule, I would suggest we add it.

```suggestion
```